pkg/codegen/python/gen.go

```diff
@@ -1809,9 +1804,10 @@ func (mod *modContext) genFunction(fun *schema.Function) (string, error) {
 			if i > 0 {
 				fmt.Fprintf(w, ",")
 			}
-			// Use the get_dict_value utility instead of calling __ret__.get directly in case the __ret__
-			// object has a get property that masks the underlying dict subclass's get method.
-			fmt.Fprintf(w, "\n        %[1]s=__ret__.%[1]s", PyName(ret.Name))
+			// Use the `pulumi.get()` utility instead of calling `__ret__.get` directly in case the `__ret__`
+			// object defines a get property with behavior other than simply returning the property value.
+			// `pulumi.get()` can only be used with classes decorated with `@input_type` or `@output_type`
+			fmt.Fprintf(w, "\n        %[1]s=pulumi.get(__ret__, '%[1]s')", PyName(ret.Name))
```

Comment:
Why has this changed to `pulumi.get`? Seems unrelated to the deprecated change.

Reply:
It's based on the suggestion in the initial issue.

This sidesteps the `getter` function which causes instantiation to print deprecation messages that we're trying to hide from the user.

I'll update the comment to be more clear. :+1: